plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -90,9 +92,25 @@ global proc mayaUSDRegisterStrings() {
     register("kSaveOptionNoPromptAnn", "You can re-enable this prompt under File|Save Scene Options.");
     register("kSaveOptionTitle", "Save USD Options");
     register("kSaveOptionUnsavedEdits", "You have unsaved USD edits. How would you like to proceed?");
+    register("kUSDSelectionMode", "USD Selection Mode");
+    register("kUSDSelectionModeAnn", "Assign selection categories to prims via the Attribute Editor to activate selection modes in this menu. Choose a selection mode to reflect changes when you select prims in the Viewport and Outliner. Note: By default, the fallback selection mode is by prim. If a selection mode is applicable, it is applied to your scene but otherwise defaults to selecting by prim.");
+    register("kUSDSelectionModeAssembly", "Assembly");
+    register("kUSDSelectionModeAssemblyAnn", "Selection mode for prims set to assembly kind. Tip: Set assembly kind in the Attribute Editor to prims that are part of an important group.");
+    register("kUSDSelectionModeComponent", "Component");
```

Comment:
I wonder if these menu entries should be localized. If we show a localized string in the menu, then it won't match the actual kind metadata value stored in the prim. The metadata value will never be localized so I don't think we should localize these either. Ask Nat what she thinks.

Reply:
Okay I see below that you are using the USD tokens, such as "Kind.Tokens.model" to get the string. So these should definitely not be localized here as they won't match when using other languages.

Reply:
According to the user story, the built-in kind should start with a capital letter. This is why I had a string for them.

Reply:
I spoke with Natalia and built-in kind will not start with capital letter. So I will remove the localization.

Reply:
Okay. I would suggest using the token from USD for the menu string.